OrbitGl/CaptureWindow.cpp

```diff
@@ -924,10 +924,11 @@ void CaptureWindow::RenderHelpUi() {
     return;
   }
 
-  ImGui::Text("Start/Stop capture: 'X'");
-  ImGui::Text("Time zoom: scroll or CTRL+right-click/drag");
+  ImGui::Text("Start/Stop capture: 'x'");
+  ImGui::Text("Pan/Zoom in capture: 'a','s','w','d'");
```

Comment:
Would switch order to WASD, as that is what most people will be familiar with (mostly from games).

Reply:
done